build-system/tasks/runtime-test.js

```diff
@@ -134,7 +135,9 @@ function printArgvMessages() {
     unit: 'Running only the unit tests. Requires ' +
         cyan('gulp css') +  ' to have been run first.',
     randomize: 'Randomizing the order in which tests are run.',
-    testlist: 'Running the tests listed in ' + cyan(argv.testlist),
+    a4a: 'Runs all A4A tests',
```

Comment:
This message is printed during test runs to indicate what is being done in the current test run. Change to

```
'Running all A4A tests.'
```

Reply:
Or `Running only A4A tests.`, if that is more accurate.